apps/src/templates/currentUserRedux.js

```diff
@@ -31,13 +32,18 @@ export const setInitialData = serverUser => ({
   type: SET_INITIAL_DATA,
   serverUser
 });
+export const setMuteMusic = isBackgroundMusicMuted => ({
```

Comment:
If it's already being passed from the backend in another place, let's set that value to redux instead of setting it from the API response.

Reply:
@maureensturgeon Do we want to use redux here because that's the pattern in TopInstructions?  If it were possible to get the needed data to this component without redux, would that be better or worse?

Reply:
I think what we're trying to avoid by using redux is the prop drilling, there's a lot of room for error with prop drilling and it also is harder to follow

Reply:
If ya'll want to explore what it would take to pass the component data without redux, I definitely think it's worth exploring that though!